tests/modules/layers/test_normalizations.py

```diff
@@ -20,3 +28,45 @@ def test_fp32groupnorm():
     norm = Fp32GroupNorm(2, 4)
     output = norm(x)
     assert output.dtype == torch.float16
+
+
+def test_rms_norm_fp32return():
+    """verify type is returned as fp32"""
+    dims = 512
+    x = torch.empty(dims, dtype=torch.float16)
+    norm = RMSNorm(
+        dims,
+    )
+    output = norm(x)
+    assert output.dtype == torch.float32
+
+
+@gpu_test(1)
```

Comment:
why is gpu required? for fp16 support?

Reply:
it's not required per se - however for finer precision / lower precision dtypes I have seen times where the PT cpu version impl does not match the gpu version impl.
Since we care about gpu, figured it's better to run this test on gpu. 
Can remove this if it creates to much test overhead?

Reply:
removed the gpu decorator since CI does not necessarily have gpu.